keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/actuation/ArtifactPromotionListener.kt

```diff
@@ -1,48 +0,0 @@
-package com.netflix.spinnaker.keel.ec2.actuation
-
-import com.netflix.spinnaker.keel.api.ec2.ArtifactImageProvider
-import com.netflix.spinnaker.keel.api.ec2.ClusterSpec
-import com.netflix.spinnaker.keel.api.ec2.ServerGroup
-import com.netflix.spinnaker.keel.clouddriver.CloudDriverService
-import com.netflix.spinnaker.keel.events.ResourceDeltaResolved
-import com.netflix.spinnaker.keel.persistence.ArtifactRepository
-import com.netflix.spinnaker.keel.persistence.DeliveryConfigRepository
-import org.slf4j.LoggerFactory
-import org.springframework.boot.autoconfigure.condition.ConditionalOnBean
-import org.springframework.context.event.EventListener
-import org.springframework.stereotype.Component
-
-@Component
-@ConditionalOnBean(CloudDriverService::class)
-class ArtifactPromotionListener(
```

Comment:
No longer using this for promotion, we'll just wait until the next resource check picks it up.